opentelemetry-api/src/opentelemetry/environment_variables/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+OTEL_PROPAGATORS = "OTEL_PROPAGATORS"
+OTEL_PYTHON_CONTEXT = "OTEL_PYTHON_CONTEXT"
+OTEL_PYTHON_DISABLED_INSTRUMENTATIONS = "OTEL_PYTHON_DISABLED_INSTRUMENTATIONS"
+OTEL_PYTHON_IDS_GENERATOR = "OTEL_PYTHON_IDS_GENERATOR"
+OTEL_PYTHON_SERVICE_NAME = "OTEL_PYTHON_SERVICE_NAME"
+OTEL_PYTHON_EXPORTER = "OTEL_PYTHON_EXPORTER"
```

Comment:
are any of these required to be in the API? Also note that the exporter env variable has been changed to `OTEL_TRACE_EXPORTER` and `OTEL_METRICS_EXPORTER`: https://github.com/open-telemetry/opentelemetry-specification/pull/1318

Reply:
I am keeping these in the API that is the common package for all the places where these environment variables are being used. We can either create a new package just for environment variables or create a module in every package for its environment variables.

Reply:
Removed `OTEL_PYTHON_EXPORTER`